### ISSUE TITLE
Fix installation steps

### ISSUE THREAD

**Issue by payne-chris-r** [ga-wdi-boston/browser-template#101]
We should have "For developers" and "For instructors" `How-to-browser-template` sections.

**Comment by gaand** [comment]
Did you check existing issues ❓ 

#98 ❓

**Comment by payne-chris-r** [comment]
No. I just wanted to get it recorded somewhere while we were talking about it

**Comment by gaand** [comment]
But you could have improved where it was already recorded.
